ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/NettyPerfClient.java

```diff
@@ -624,8 +621,12 @@ private void reset() {
         request.headers().add(RestUtils.Headers.BLOB_SIZE, totalSize);
         request.headers().add(RestUtils.Headers.SERVICE_ID, serviceId);
         request.headers().add(RestUtils.Headers.AMBRY_CONTENT_TYPE, "application/octet-stream");
-        request.headers().add(RestUtils.Headers.TARGET_ACCOUNT_NAME, targetAccountName);
-        request.headers().add(RestUtils.Headers.TARGET_CONTAINER_NAME, targetContainerName);
+        if (targetAccountName != null) {
```

Comment:
Do you think it's worthwhile to check `!targetAccountName.isEmpty()` as well?
Same for container's check.

Reply:
Thanks for the advice. Yes, I should add isEmpty check for account/container. I've update the code.